front/src/components/ProfileView/ProfileBanner.tsx

```diff
@@ -90,32 +91,39 @@ const ProfileBanner: React.FC<IProfileBannerProps> = ({
           )}
         </div>
       )}
-      <div className="flex flex-row flex-nowrap items-center pt-3 pb-3">
+      <div className="flex flex-row flex-nowrap items-start pt-3 pb-3">
         <Skeleton visible={loading} w="5.5rem" radius="md">
-          <ProfilePicture
-            src={
-              loading
-                ? null
-                : GigzFetcher.getImageUri(profilePicture?.media as string)
-            }
-            radius="xl"
-            size="xl"
-            alt={username}
-          />
+          {editMode.editMode ? (
```

Comment:
editMode.editMode is a little strange, editMode.isEditing or something like that would, I think, be better